.gitignore

```diff
@@ -0,0 +1 @@
+/.vscode
```

Comment:
This folder is often added to git, to share vscode-specific settings that other team members use. However, as this is a solo project, we can ignore it.

Reply:
@hhu-wtag  vai suggested [removing](https://github.com/ban-wtag/ToDoAppJS/pull/1#discussion_r1161207957) it from the project.